src/main/kotlin/org/rust/ide/typing/paste/RsConvertJsonToStructCopyPasteProcessor.kt

```diff
@@ -21,6 +21,7 @@ import com.intellij.psi.PsiDocumentManager
 import com.intellij.psi.PsiElement
 import com.intellij.psi.PsiFile
 import com.intellij.psi.SmartPsiElementPointer
+import com.jetbrains.rd.util.first
```

Comment:
Let's try to avoid using `com.jetbrains.rd` library not to create additional dependency on some platform dependency without actual necessity.

Sometimes, it leads to quite unexpected consequences like `NoClassDefFoundError` in some IDEs that doesn't contain the lib, or problems with compilation with new platform version where bundled library set is changed, etc.

Reply:
Good catch, this was a wild autoimport I guess.